students/jemielity_kamil/lesson_06_dicts_tuples_sets_args_kwargs/guess_the_number.py

```diff
@@ -0,0 +1,17 @@
+
+n = int(input('Number: '))
+right_set = set(range(1, n + 1))
+wrong_set = set()
+
+while True:
+    numbers = input('Set of numbers: ')
+    if numbers != 'HELP':
+        numbers = set(numbers.split())
+        bob_answer = input('Bob answer: ')
+        if bob_answer == 'YES':
+            right_set.intersection_update(numbers)
+        else:
+            wrong_set.update(numbers)
```

Comment:
The algorithm is correct although a few comments from my side ;)

Instead of splitting sets to right and wrong we can create common one like in the code below

```
n = int(input('Number n: '))
possible_set = set(range(1, n + 1))

while True:
    beatrice_input = input('Beatrice set of numbers: ')
    if beatrice_input == 'HELP':
        break
    beatrice_input = set(int(a) for a in beatrice_input.split())
    bob_answer = input('Bob answer: ')
    if bob_answer == 'YES':
        possible_set.intersection_update(beatrice_input)
    else:
        possible_set.difference_update(beatrice_input)

print(" ".join(str(s) for s in sorted(possible_set)))
```